app/livechat/client/views/regular.js

```diff
@@ -8,6 +8,7 @@ import './app/tabbar/agentInfo';
 import './app/tabbar/externalSearch';
 import './app/tabbar/visitorEdit';
 import './app/tabbar/visitorForward';
-import './app/tabbar/visitorHistory';
+import './app/tabbar/customerChatHistory';
```

Comment:
If you're replacing the old template you'll need to remove the old file as well.

Reply:
i have removed the file as well